### PR TITLE
chores(TDP-3541): dataset parameters panel improvements

### PR DESCRIPTION
**Pull request by ncomont**
**Link to the JIRA issue**
https://jira.talendforge.org/browse/TDP-3541

**Please check if the PR fulfills these requirements**
- [x] The commit(s) message(s) follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format)
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] The code coverage on new code is > 75 % for backend and > 95% for frontend
- [x] The new code does not introduce new technical issues (sonar / eslint)
- [x] Functional tests have been performed
- [ ] Docker configuration files for config-std or config-cloud profiles are impacted

**Please check the browsers you've tested on**
- [x] Chrome, Firefox, Safari, Edge, IE11
- [ ] No, that's bad, this PR should not be merged !
- [ ] No, and no need to (backend changes only)

**(Optional) What is the current behavior?**
(Additional information to the Jira)

**(Optional) What is the new behavior?**
(Additional information to the Jira)

**(Optional) Other information**:

### CODE REVIEW

**Comment by build-talend-tdp** [comment]
Pull request test result: [failure](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/313/) [:x:](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/313/)

**Comment by build-talend-tdp** [comment]
Pull request test result: [failure](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/314/) [:x:](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/314/)

**Comment by build-talend-tdp** [comment]
Pull request test result: [failure](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/316/) [:x:](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/316/)

**Comment by build-talend-tdp** [comment]
Pull request test result: [failure](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/317/) [:x:](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/317/)

**Comment by build-talend-tdp** [comment]
Pull request test result: [failure](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/318/) [:x:](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/318/)

File: dataprep-webapp/src/app/components/dataset/parameters/_dataset-parameters.scss [hunk 1]
```diff
@@ -12,84 +12,52 @@
...
 
-			.param-row-item {
-				display: flex;
-				flex-direction: row;
-				flex-wrap: wrap;
-				align-items: center;
-				flex-basis: 300px;
-				flex-grow: 1;
+			.form-group {
+				flex-basis: 50%;
```
Comment (amaalej) [inline dataprep-webapp/src/app/components/dataset/parameters/_dataset-parameters.scss]: In 1360X768 resolution, we have this result
resolution1360

File: dataprep-webapp/src/app/components/dataset/parameters/dataset-parameters-directive.spec.js [hunk 2]
```diff
@@ -25,8 +25,8 @@ describe('Dataset parameters directive', function () {
             DATASET_PARAMETERS_SEPARATOR: 'Separator',
             FILE_DETAILS_LINES: '{{records}} lines',
             FILE_DETAILS_LIMIT: 'cut at {{records}} lines',
-            NAME: 'Name :',
-            SIZE: 'Size :',
+            NAME: 'Name',
```
Comment (amaalej) [inline dataprep-webapp/src/app/components/dataset/parameters/dataset-parameters-directive.spec.js]: I recommend you to use ```const i18en = {... : ...}``` then refer to the different values

File: dataprep-webapp/src/app/components/dataset/parameters/_dataset-parameters.scss (hunk 1 above)
Comment (amaalej) [inline dataprep-webapp/src/app/components/dataset/parameters/_dataset-parameters.scss]: this is the cause of my change request.

Comment (ncomont) [inline dataprep-webapp/src/app/components/dataset/parameters/_dataset-parameters.scss]: good catch, thanks

File: dataprep-webapp/src/app/components/dataset/parameters/dataset-parameters-directive.spec.js (hunk 2 above)
Comment (ncomont) [inline dataprep-webapp/src/app/components/dataset/parameters/dataset-parameters-directive.spec.js]: fixed

**Comment by build-talend-tdp** [comment]
Pull request test result: [success](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/351/) [:star:](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/351/)

**Comment by build-talend-tdp** [comment]
Pull request test result: [success](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/352/) [:star:](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/352/)

**Comment by build-talend-tdp** [comment]
Pull request test result: [failure](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/353/) [:x:](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/353/)

**Comment by build-talend-tdp** [comment]
Pull request test result: [failure](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/354/) [:x:](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/354/)

**Pull request merged**